Dockerfile

```diff
@@ -17,13 +17,25 @@ ADD . .
 RUN make build
 
 FROM --platform=$FABRIC_BUILDER_PLATFORM $FABRIC_BUILDER_TAG AS fabric-builder
-RUN apk add libc6-compat
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    wget \
+    sudo \
+    curl \
+    ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+# Install Go
+ARG GO_VERSION=1.20
+ARG ARCH=amd64
+RUN curl -fsSL "https://golang.org/dl/go$GO_VERSION.linux-$ARCH.tar.gz" | tar -C /usr/local -xzf -
+ENV PATH="/usr/local/go/bin:$PATH"
 WORKDIR /firefly/smart_contracts/fabric/firefly-go
 ADD smart_contracts/fabric/firefly-go .
+ENV GO111MODULE=on
```

Comment:
I think this is already done in the next line. Is there a need to have it for multiple commands here? If so, should we just remove it from line 35?